cattle/plugins/docker/compute.py

```diff
@@ -90,6 +90,9 @@ def get_agent_images(self, client):
                 label_val = i['Labels'][SYSTEM_LABEL]
                 for l in i['RepoTags']:
                     system_images[l] = label_val
+                    if l.endswith(':latest'):
```

Comment:
I wonder should it just drop anything after the colon just in case it happened to be a version # instead of 'latest'